addon/lib/field.js

```diff
@@ -178,20 +180,32 @@ export default EmberObject.extend(Evented, {
    */
   save: task(function*() {
     const type = this.get("answer.__typename");
+    const value = this.get("answer.value");
 
-    return yield this.apollo.mutate(
-      {
-        mutation: this.get(`saveDocument${type}Mutation`),
+    if (value === null || value.length === 0) {
+      return yield this.apollo.mutate({
+        mutation: removeAnswerMutation,
         variables: {
           input: {
-            question: this.get("question.slug"),
-            document: this.get("document.id"),
-            value: this.get("answer.value")
+            answer: atob(this.get("answer.id")).split(":")[1]
```

Comment:
Use `import { atob } from "ember-caluma/helpers/atob";` instead

Reply:
Ok

What's the reason behind the helper?

Reply:
The helper just exists to avoid having the same logic spread across many files.